x-pack/plugins/security_solution/server/lib/detection_engine/signals/search_after_bulk_create.ts

```diff
@@ -70,6 +70,7 @@ export const searchAfterAndBulkCreate = async ({
     interval,
     buildRuleMessage,
   });
+  const tuplesToBeLogged = [...totalToFromTuples];
   logger.debug(buildRuleMessage(`totalToFromTuples: ${totalToFromTuples.length}`));
```

Comment:
Should we remove the `debug` logline since it's now covered by the `info`?

Reply:
If the execution of the big loop throws an error then the new `info` line won't get hit so I don't think removing the debug statement would be beneficial in that case.